### ISSUE TITLE
[Bug]: Server shutdown unexpectedly with code 137.

### ISSUE THREAD

**Issue by slyahoo** [httptoolkit/httptoolkit#475]
### Repro steps

Launch the app and see this error.

There is no way to recover from this crash, even tries download the dmg and replace the app again or just deleted the app, and reinstall again.

### How often does this bug happen?

Every time

### The desktop OS you're using

MacOS 13.5.2 

### Error screenshot

<img src="https://github.com/httptoolkit/httptoolkit/assets/145403290/e623bc01-d8b4-40a1-a76f-b7a1ceaaa937" width="30%" height="30%" />

### Any other info?

I am a paid (Pro version) customer. Can you give me a solution asap? thanks.

**Comment by pimterry** [comment]
Hi @slyahoo. I think this means your system is actively killing HTTP Toolkit's internal server process. This has been reported in this past as being caused by organizational security policies: https://github.com/httptoolkit/httptoolkit/issues/451#issuecomment-1638477934. Is it possible that's the issue in your case?

If not, can you share the logs from after this crash? They're available from /Users/<username>/Library/Logs/httptoolkit/last-run.log.

**Comment by slyahoo** [comment]
Hi,
This is the log.------------------------


US-PGJC3P59PK:httptoolkit SanLee$ more last-run.log 

--- Launching HTTP Toolkit desktop v1.14.3 ---

Config checked in 16 ms

Certificates setup in 10 ms

Standalone server started in 8 ms

Server started in 23 ms

Total startup took 58 ms

(node:70164) Warning: Using insecure HTTP parsing

(Use `node --trace-warnings ...` to show where the warning was created)

INFO: Initialising UI (version 9acf2b7a3e3c94e2a18db6744cf2091d24af4219)

INFO: Account store initialized

INFO: Send store initialized

INFO: UI store initialized

INFO: Proxy settings loaded

Mock session started, http on port 8000, webrtc enabled

Docker not available: connect ENOENT /var/run/docker.sock

Webextension installed at /var/folders/bt/p97_mclx0vv81kw_v5md9nlc0000gq/T/httptoolkit-webextension

INFO: API store initialized

INFO: Server started

/Applications/HTTP Toolkit.app/Contents/Resources/httptoolkit-server/bin/httptoolkit-server: line 45: 70164 Killed: 9               "$NODE" "$DIR/run" "$@"

US-PGJC3P59PK:httptoolkit SanLee$ pwd

/Users/SanLee/Library/Logs/httptoolkit

US-PGJC3P59PK:httptoolkit SanLee$ 


    On Tuesday, September 19, 2023 at 12:30:19 AM PDT, Tim Perry ***@***.***> wrote:  
 
 


Hi @slyahoo. I think this means your system is actively killing HTTP Toolkit's internal server process. This has been reported in this past as being caused by organizational security policies: #451 (comment). Is it possible that's the issue in your case?

If not, can you share the logs from after this crash? They're available from /Users//Library/Logs/httptoolkit/last-run.log.

**Comment by pimterry** [comment]
Yes, I think that confirms this - something on your computer is intentionally shutting down HTTP Toolkit.

You'll need to investigate that I'm afraid. This could be some kind of security policy, or a lack of memory available (in which case the OS will stop the program), or some other background process that's managing & closing processes, it's hard to say I'm afraid.
